### PR TITLE
Updating feedback colours

### PR DESCRIPTION
**Pull request by gjones**
### Summary
This PR updates the feedback colours to be less striking and slightly more muted on both dark and light themes. 

This effects the following components
- Alerts
- Badges
- Button (Danger)

#### Examples
CleanShot 2024-07-19 at 17 06 27@2x
CleanShot 2024-07-19 at 17 09 03@2x
CleanShot 2024-07-19 at 17 08 47@2x

### CODE REVIEW

**Review by crisalbu — COMMENTED** [review]
Great to see a PR for this @gjones ! 🙌🏻 

I've noticed that the border for the danger button is darker than the one for badges (10% opacity instead on 5%) - is that on purpose?

Since this token change impacts buttons as well, we should also define translucent colours for hover and active states. I was thinking we could do this - see `now` vs `proposed change`:

Screenshot 2024-07-22 at 10 23 25


Screenshot 2024-07-22 at 10 03 38

For the active state, I've also changed the text colour in light mode.

[Designs can be found here](https://www.figma.com/design/mjSci5n5YREyVlvaVjMSDq/Click-UI-Library?node-id=8655-5820&t=2agzVigSRl3d4YI6-1)

**Comment by gjones** [comment]
@crisalbu updated based on feedback

https://github.com/user-attachments/assets/f1c674f7-e878-4178-a1af-7715273357ab

**Comment by crisalbu** [comment]
Is there anywhere I can test these? I was looking in Figma and noticed we're missing the icon background, which I'd make the same as the general background.

Screenshot 2024-07-22 at 19 14 25

**Comment by gjones** [comment]
> Is there anywhere I can test these?

See the visit preview link in the generated comment above.
Yeah the Figma is put together differently that the code, in Figma the left and the right are two separate areas, in code the left area sits on top of the right area so there is no gap in colour. I was going to fix the Figma but haven't gotten around to it.

**Review by crisalbu — APPROVED** [review]
Thanks, good to know ! :) 

Just tested and it looks great, thank you so much @gjones! ❤️ 🎉 

We might want to tackle the default one, especially the border, but we can do it at a later time.

File: src/styles/variables.dark.json [hunk 1]
```diff
@@ -48,11 +48,11 @@
         },
         "iconBackground": {
           "default": "#1F1F1C",
-          "success": "#004206",
-          "neutral": "#414141",
-          "danger": "#610000",
-          "warning": "#4f2b00",
-          "info": "#09255B"
+          "success": "rgb(20% 100% 26.7% / 0)",
```
[the remark this answers is not in the document]
Comment (rndD) [reply src/styles/variables.dark.json:51]: related bug https://github.com/ClickHouse/control-plane/issues/10026

**Pull request merged**